airiam/find_unused/RuntimeIamScanner.py

```diff
@@ -12,6 +14,13 @@
 ERASE_LINE = '\x1b[2K'
 
 
+def get_iam_data_file(account_id=None):
```

Comment:
@manishsingh10895 Please remove the default `None` value (enforces supplying a value) and denote it as an str like so:
```suggestion
def get_iam_data_file(account_id: str):
```